lib/smart_answer/calculators/check_travel_during_coronavirus_calculator.rb

```diff
@@ -22,6 +22,15 @@ def country_locations
       end
     end
 
+    def summary_text_fields
+      return [] if travelling_to_ireland? && single_journey?
+
+      @summary_text_fields = @travelling_with_children.dup
```

Comment:
So I figured the `@travelling_with_children` object was a good place to start because it's an array that has some of the values I want and the only place where there can currently be multiple values that I'd need to loop through. The `.dup` is needed because without it you end up with that horrible problem I had on Friday where `fully_vaxed` was added everywhere

Reply:
I think that was because you were assigning to the instance variable directly, rather than calling the method?

i.e. What happens if you try `@summary_text_fields = travelling_with_children`. (I'm still not sure why `@summary_text_fields` is an instance method.

Reply:
yeah, have fixed all of this, silly error, sorry!